psvg.ts

```diff
@@ -440,12 +449,4 @@ export function compilePSVG(psvg:string):any{
   return evalPSVG(js);
 }
 
-if (typeof window !== 'undefined') {
-  window.addEventListener('load',function(){
-    const psvgs = document.getElementsByTagName("PSVG");
-    for (let i = 0; i < psvgs.length; i++){
-      psvgs[i].outerHTML = compilePSVG(psvgs[i].outerHTML);
-    }
-  })
-}
```

Comment:
Wondering why is this removed?

Reply:
I did not edit that file directly; when I used the `npm run build` command, it was overwritten on its own.

Reply:
Due to my operational error it seems to have removed.

Reply:
I've fixed it.